common/vpcclient/models/security_group.go

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * IBM Confidential
+ * OCO Source Materials
+ * IBM Cloud Kubernetes Service, 5737-D43
+ * (C) Copyright IBM Corp. 2023 All Rights Reserved.
+ * The source code for this program is not published or otherwise divested of
+ * its trade secrets, irrespective of what has been deposited with
+ * the U.S. Copyright Office.
+ ******************************************************************************/
+
+// Package models ...
+package models
+
+import (
+	"github.com/IBM/ibmcloud-volume-interface/lib/provider"
+)
+
+// SecurityGroup ...
+type SecurityGroup struct {
+	CRN           string         `json:"crn,omitempty"`
+	Href          string         `json:"href,omitempty"`
+	ID            string         `json:"id,omitempty"`
```

Comment:
Please check omitempty for ID is really make sense

Reply:
We will verify this in next PR https://github.ibm.com/alchemy-containers/armada-storage/issues/4583